### DISCUSSION TITLE
Does AR.js use ARKit when running on iOS?

### DISCUSSION THREAD

**Discussion by axelson** [AR-js-org/AR.js#299] [Help] [answered]
I looked for an answer to this in the docs and webpage but I cannot tell for certain. From this issue it looks like ARKit is not used? https://github.com/AR-js-org/AR.js/issues/217

Since the current last message says "Adding support to (ARcore, ARKit...) would be nice but not a prority."

**Comment by kalwalt** [comment, marked as answer]
HI @axelson welcome to AR.js! I can confirm to you that **AR.js** doesn't not support ARKit or ARcore, I think that would be possible but we have other priorities, see for example **ES6** feature https://github.com/AR-js-org/AR.js/pull/116.

**Reply by axelson** [reply]
Okay, thanks :+1:
